doc/data/messages/b/bad-exception-context/bad.py

```diff
@@ -0,0 +1,7 @@
+def divide(x, y):
+    result = 0
+    try:
+        result = x / y
+    except ZeroDivisionError:
+        raise Exception("Can't divide by zero!") from result  # [bad-exception-context]
```

Comment:
```suggestion
    except ZeroDivisionError:
        raise ValueError(f"Division by zero when dividing {x} by {y} !") from result  # [bad-exception-context]
```

Reply:
Yes, I was also a bit hesitant using just the general ``Exception`` class. Changed it. Thanks for the feedback!

Reply:
Yeah, I think it's bad style to re-reaise an exception that is more generic than the one caught which is why we dont like this, but ValueError do not fix that. Let me suggest something with a custom exception.